### PR TITLE
Fix 942

### PR DESCRIPTION
**Pull request by dkochmanski**
This PR fixes the issue with duplicated definitions.

### CODE REVIEW

File: Backends/CLX/medium-xrender.lisp [hunk 1]
```diff
@@ -6,7 +6,8 @@
 (in-package #:clim-clx)
```
Comment (scymtym) [inline Backends/CLX/medium-xrender.lisp]: Commit message: "backens" -> "backend" or "backends".

File: Core/clim-basic/decls.lisp [hunk 2]
```diff
@@ -1201,11 +1203,6 @@ rendered on MEDIUM with the style LINE-STYLE."))
 
 ;;;
 
-(defgeneric text-style-character-width (text-style medium char))
-;; fall back, where to put this?
-(defmethod text-style-character-width (text-style medium char)
-  (text-size medium char :text-style text-style))
-
```
Comment (scymtym) [inline Core/clim-basic/decls.lisp]: I'm not sure this change is an improvement. I usually consider unspecialized default methods as part of the protocol. I don't have a super strong opinion on this, though.

Comment (dkochmanski) [reply Core/clim-basic/decls.lisp]: fixed (I've put it in :method clause of defgeneric to signify the fact that it is part of this protocol function). n.b it is an internal protocol which is expected to be implemented by backends for purpose of extended output stream width specifications (i.e there is no clim:text-style-character-width).

File: Backends/CLX/medium-xrender.lisp (hunk 1 above)
Comment (dkochmanski) [reply Backends/CLX/medium-xrender.lisp]: fixed

**Pull request merged**